### DISCUSSION TITLE
Add Quake mode

### DISCUSSION THREAD

**Discussion by RootMePLS** [ghostty-org/ghostty#3894] [Feature Requests, Ideas] [answered]
Quake mode allows the terminal to be toggled quickly with a keyboard shortcut, sliding down from the top of the screen. 
 his functionality is particularly useful for users who need to access the terminal frequently but want to maintain a clutter-free workspace. It enhances productivity by providing instant access to the terminal without disrupting the current workflow.

**Comment by JakeWharton** [comment, marked as answer]
This already exists (macOS only for now). See https://ghostty.org/docs/features#macos (or the many other discussions and issues about its capabilities).

**Reply by RootMePLS** [reply]
Just what I needed, thank you!
I suggest adding the word "Quake" to the documentation page, as this term speaks for itself.

**Reply by hedgieinsocks** [reply]
In the meantime, gnome users might try https://extensions.gnome.org/extension/6063/toggler/ instead.

**Comment by andremacola** [comment]
It still needs to be improved, support for tabs for example.

**Reply by rhodes-b** [reply]
> It still needs to be improved, support for tabs for example.

Please look for other discussions on why it doesnt support tabs
https://github.com/ghostty-org/ghostty/discussions/2888
